### PR TITLE
Mooovers homepage

### PR DESCRIPTION
**Pull request by joyjoie**
#### What does this PR do?
Allows Users to view Homepage

#### Description of Task to be completed?
- [x] Adding Navbar to Homepage
-  [x] Adding of Homepage
-  [x] Updating background image and links of the Homepage
-  [x] Updating style of Homepage

#### Screenshots
Screenshot 2019-10-01 at 00 30 46

Screenshot 2019-10-01 at 00 33 43

### CODE REVIEW

File: src/components/HomePage/footer.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      <p>Terms of Use</p>
+    </div>
+    <div class="column">
+      <p>Transporting Companies</p>
+      <p>How It Works</p>
+    </div>
+    <div class="column">
+      <p>Email:info@mooovers.com</p>
+      <p> All Rights reserved</p>
+      <hr></hr>
```
Comment (Nicanor008) [inline src/components/HomePage/footer.js]: use one hr here like this `<hr />`. using self closing tag is helpful to tags not taking any children

File: src/components/HomePage/footer.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <p>@2019</p>
```
Comment (Nicanor008) [inline src/components/HomePage/footer.js]: the current year can be autogenerated, see more here,  http://updateyourfooter.com

File: src/components/HomePage/home.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  Header, 
+  Image,
+  List,
+  Button,
+  Segment,
+} from 'semantic-ui-react'
+
+const Home = () => (
+  <div className="homepage">
+
```
Comment (Nicanor008) [inline src/components/HomePage/home.js]: empty spaces with in, kindly add a comment why this should be there if it should

File: src/components/HomePage/home.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        <Grid.Column>
+          <h2>Platform Provision</h2>
+          <p>Platform for users moving items and companies giving out Goods and Transportation Services.</p>
+        </Grid.Column>
+        <Grid.Column>
+        <h2>Moving Services</h2>
+        <p>We move all types of goods across Cities, States and Countries.</p>
+        </Grid.Column>
+        <Grid.Column>
+          <h2>Mananged Services</h2>
```
Comment (Nicanor008) [inline src/components/HomePage/home.js]: managed services not mananged services

File: src/components/HomePage/home.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+          <h2>HOW IT WORKS</h2>
+          <p>Platform for users moving items and companies giving out Goods and Transportation Services.</p>
+        </Grid.Column>
+        <Grid.Column>
+        <Image src={Map} id="map"/>
+        </Grid.Column>
+      </Grid.Row>
+      </Grid>
+    </Container>
+    <Segment inverted vertical style={{ margin: '5em 0em 0em', padding: '5em 0em' }}>
```
Comment (Nicanor008) [inline src/components/HomePage/home.js]: remove the inline css

File: src/components/MenuLayout.scss [hunk 6]
```diff
@@ -51,3 +51,94 @@
...
+}
+
+.header{
+    text-align: center;
+}
+
+.location{
+    margin-left: auto;
+    margin-right: 2px;
+
```
Comment (Nicanor008) [inline src/components/MenuLayout.scss]: empty spaces

File: src/components/HomePage/footer.js (hunk 1 above)
Comment (joyjoie) [reply src/components/HomePage/footer.js]: modified

File: src/components/HomePage/home.js (hunk 4 above)
Comment (joyjoie) [reply src/components/HomePage/home.js]: modified

File: src/components/HomePage/home.js (hunk 5 above)
Comment (joyjoie) [reply src/components/HomePage/home.js]: updated changes

File: src/components/MenuLayout.scss (hunk 6 above)
Comment (joyjoie) [reply src/components/MenuLayout.scss]: modified

File: src/components/HomePage/footer.js (hunk 2 above)
Comment (joyjoie) [reply src/components/HomePage/footer.js]: Done

File: src/components/MenuLayout.js [hunk 7]
```diff
@@ -7,6 +7,8 @@ import logo2 from '../assets/images/logo2.png';
 import { Container, Image, Menu, Dropdown } from "semantic-ui-react";
 import Register from './auth/register/register';
 import Login from './auth/login/login';
+// import Home from './HomePage/home';
```
Comment (Nicanor008) [inline src/components/MenuLayout.js]: remove the comment

Comment (joyjoie) [reply src/components/MenuLayout.js]: resolved

**Pull request merged**